### ISSUE TITLE
2020-04-26 09:32:04.667319: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:233] Failed to run optimizer ArithmeticOptimizer, stage HoistCommonFactor. Error: Node yolo_loss/ArithmeticOptimizer/HoistCommonFactor_Add_add_20 is missing output properties at position :0 (num_outputs=0)

### ISSUE THREAD

**Issue by chenxiao521** [Ma-Dan/keras-yolo4#1]
您好，请问一下，我训练的时候出现这样的提示，会影响训练结果吗，是什么原因导致的呢。

**Comment by Ma-Dan** [comment]
请使用刚才更新的代码进行训练
https://github.com/Ma-Dan/keras-yolo4/commit/6bf056ae7f46a27df80fa3ffe873c69cc6478b86
我测试的环境：tensorflow 1.14.0， keras 2.2.4
目前代码用的是DIoU loss，可以训练出结果。

**Comment by wenchao1993** [comment]
你好，我也遇到了同样的问题，请问这个问题解决了吗？

**Comment by wenchao1993** [comment]
> 您好，请问一下，我训练的时候出现这样的提示，会影响训练结果吗，是什么原因导致的呢。

你好，请问该问题解决了么？

**Comment by c464851257** [comment]
你好，我也遇到了同样的问题，请问这个问题解决了吗？

**Comment by Ma-Dan** [comment]
我测试的环境tensorflow 1.14.0，Keras 2.2.4

**Comment by c464851257** [comment]
好的 谢谢 我试一试
